yogstation/code/modules/antagonists/darkspawn/darkspawn_abilities/scout_abilities.dm

```diff
@@ -0,0 +1,284 @@
+//////////////////////////////////////////////////////////////////////////
+//----------------------Scout light eater ability-----------------------//
+//////////////////////////////////////////////////////////////////////////
+/datum/action/cooldown/spell/toggle/light_eater
+	name = "Light Eater"
+	desc = "Twists an active arm into a blade of all-consuming shadow."
+	panel = "Darkspawn"
+	button_icon = 'yogstation/icons/mob/actions/actions_darkspawn.dmi'
+	background_icon_state = "bg_alien"
+	overlay_icon_state = "bg_alien_border"
+	buttontooltipstyle = "alien"
+	button_icon_state = "light_eater"
+	check_flags = AB_CHECK_HANDS_BLOCKED | AB_CHECK_CONSCIOUS | AB_CHECK_LYING
+	spell_requirements = SPELL_REQUIRES_DARKSPAWN | SPELL_REQUIRES_HUMAN
+
+/datum/action/cooldown/spell/toggle/light_eater/process()
+	active = owner.is_holding_item_of_type(/obj/item/light_eater)
+	. = ..()
+
+/datum/action/cooldown/spell/toggle/light_eater/can_cast_spell(feedback)
+	if(!owner.get_empty_held_indexes() && !active)
+		if(feedback)
+			to_chat(owner, span_warning("You need an empty hand for this!"))
+		return FALSE
+	. = ..()
+
+/datum/action/cooldown/spell/toggle/light_eater/Enable()
+	to_chat(owner, span_velvet("Akna"))
```

Comment:
This barely visible single 4 letter word of text is completely useless to the user, especially since the line right under gives actual useful information. At the very least make it a balloon alert so it isnt lingering in the chat. Same for disable.

Reply:
lose stuff